### PR TITLE
Added automation tests incorporating the recent changes by Approved Premises Delius ==> OASys Layer3 ==> AP Applications ==> Approved Premises - Create a Booking ==> Delius verification

### PR DESCRIPTION
**Pull request by smalepati99**
Given I login in to NDelius
And I create an offender and event in Delius
And I create a Layer 3 Assessment without Needs in OASys
And I login to Approved Premises
And I navigate to Approved Premises - Applications
And I complete all the sections and submit the application for this CRN
And I log back to Approved Premises
And I navigate to create a placement # Choose Actions > Create a placement
When I create a booking in Approved Premises
Then I should see the staff member in the list of Key Workers
And login to nDelius
And I Search for offender with CRN
And I should see a contact in Delius for the booking

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ const config: PlaywrightTestConfig = {
             name: 'chromium',
             use: {
                 ...devices['Desktop Chrome'],
+                viewport: {width: 1792, height: 886},
```
Comment (anthony-britton-moj) [inline playwright.config.ts]: is this required for the tests ?

File: steps/approved-premises/applications/access-cultural-healthcare-needs.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { type Page, expect } from '@playwright/test'
+
+export const addAccessCulturalHealthCareNeeds = async (page: Page) => {
+    await page.locator('#additionalNeeds').check()
+    await page.locator('#religiousOrCulturalNeeds-2').check()
+    await page.locator('#needsInterpreter-2').check()
+    await page.locator('#careActAssessmentCompleted-2').check()
+    await page.locator('.govuk-button', { hasText: 'Save and continue' }).click();
+    //"Specify any additional details and the adjustments required" Page
```
Comment (anthony-britton-moj) [inline steps/approved-premises/applications/access-cultural-healthcare-needs.ts]: a space between comment indicators and the comment might be more consistent? not sure if the linter will complain

File: steps/approved-premises/applications/confirm-details.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { type Page, expect } from '@playwright/test'
 
 export const clickSaveAndContinue = async (page: Page) => {
-    await page.locator('button', { hasText: 'Save and continue' }).click()
+    await page.locator('.govuk-button',{ hasText: 'Save and continue' }).click();
```
Comment (anthony-britton-moj) [inline steps/approved-premises/applications/confirm-details.ts]: not sure we need semi colons ... linter may complain

File: steps/approved-premises/applications/check-your-answers.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { type Page, expect } from '@playwright/test'
+
+export const checkYourAnswers = async (page: Page) => {
+    await page.keyboard.down('End')
+    await page.locator('button', { hasText: 'Continue' }).click();
```
Comment (anthony-britton-moj) [inline steps/approved-premises/applications/check-your-answers.ts]: semi-colon?

File: steps/approved-premises/applications/attach-required-documents.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import { type Page, expect } from '@playwright/test'
+
+export const attachReqrdDocuments = async (page: Page) => {
+    await page.locator('.govuk-button',{ hasText: 'Save and continue' }).click();
```
Comment (anthony-britton-moj) [inline steps/approved-premises/applications/attach-required-documents.ts]: copy and past of semi-colon 😆

File: steps/delius/utils/date-time.ts [hunk 6]
```diff
@@ -8,6 +8,7 @@ export const EuropeLondonFormat = (date: Date) =>
 
 //Getting dates using date-fns lib
 export const NextMonth = addMonths(new Date(), 1)
+export const AfterFiveMonths = addMonths(new Date(), 5)
```
Comment (anthony-britton-moj) [inline steps/delius/utils/date-time.ts]: addMonths(new Date(), 5) is probably short enough ... will we use 5 months regularly?

**Review by anthony-britton-moj — DISMISSED** [review]

File: playwright.config.ts (hunk 1 above)
Comment (smalepati99) [reply playwright.config.ts]: It is now removed

File: steps/approved-premises/applications/access-cultural-healthcare-needs.ts (hunk 2 above)
Comment (smalepati99) [reply steps/approved-premises/applications/access-cultural-healthcare-needs.ts]: Added spaces

File: steps/approved-premises/applications/confirm-details.ts (hunk 3 above)
Comment (smalepati99) [reply steps/approved-premises/applications/confirm-details.ts]: Semi colons are now removed for all the statements, its possibly a paste issue

File: steps/approved-premises/applications/check-your-answers.ts (hunk 4 above)
Comment (smalepati99) [reply steps/approved-premises/applications/check-your-answers.ts]: Semi colons are now removed for all the statements, its possibly a paste issue

File: steps/approved-premises/applications/attach-required-documents.ts (hunk 5 above)
Comment (smalepati99) [reply steps/approved-premises/applications/attach-required-documents.ts]: Semi colons are now removed for all the statements, its possibly a paste issue

File: steps/delius/utils/date-time.ts (hunk 6 above)
Comment (smalepati99) [reply steps/delius/utils/date-time.ts]: It is now changed

**Review by anthony-britton-moj — APPROVED** [review]

**Pull request merged**